packages/adblocker-puppeteer/test/adblocker.test.ts

```diff
@@ -36,3 +39,53 @@ describe('#fromPuppeteerDetails', () => {
     });
   });
 });
+
+describe('#stylesInjection', () => {
+  let server: Server;
+  let port: number;
+  let browser: puppeteer.Browser;
+  let page: puppeteer.Page;
+
+  before(async () => {
+    const app = express();
+    app.get('/', (_req: any, res: any) => {
+      res.send('<html><title>Empty test HTML</title></html>');
+    });
+    server = app.listen(0, () => {
```

Comment:
Will this pick an available port?

Reply:
Yes, this is done exactly for that purpose :)

Reply:
Nice, I did not know this. Is this documented somewhere?

Reply:
http://expressjs.com/en/5x/api.html#app.listen
"If port is omitted or is 0, the operating system will assign an arbitrary unused port, which is useful for cases like automated tasks (tests, etc.)."

Reply:
Very nice, I overlooked this part of the doc. Thanks!